wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -190,7 +190,7 @@ public function addUserToCollection($data): WP_REST_Response|false
             return new WP_REST_Response([
                 [
                     "status" => $statusCode,
-                    "message" => $email . __(" was added to the Collection."),
+                    "message" => $email . __(" was invited to the Collection."),
```

Comment:
This is a good catch. 

Although in the users table, they appear right away. Maybe we should add a flag somewhere that they are still just "invited".

Reply:
Ya - it's not accurate as far as the users table goes + real status as the account is added but makes sense from a user entering the data end.

Reply:
Would could add a flag (in another PR) to see if they've ever logged in ... starts to move into an update after your PR is merged https://github.com/cds-snc/gc-articles/pull/217

i.e. check for never logged in.